junction/proposals/models.py

```diff
@@ -59,6 +59,19 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
+class ProposalStatus(AuditModel):
+
+    """ List of Proposal Types """
+    name = models.CharField(max_length=255, verbose_name="Proposal Status Name")
+    description = models.TextField(default="")
+    active = models.BooleanField(default=True, verbose_name="Is Active?")
+    conferences = models.ManyToManyField(to=Conference, related_name='proposal_status')
+
+    def __str__(self):
+        return self.name
```

Comment:
Why do we need this model? If we wan to know, if the proposal is reviewed or not, we can just add a new property `is_reviewed` to model to `Proposal` model class itself. Thoughts?